packages/docmanager/src/manager.ts

```diff
@@ -584,6 +584,21 @@ export class DocumentManager implements IDocumentManager {
       return undefined;
     }
 
+    let fileType: DocumentRegistry.IFileType | undefined = undefined;
+    for (const ft of this.registry.fileTypes()) {
+      if (ft.name === modelName) {
+        fileType = ft;
+        break;
+      }
+    }
```

Comment:
@davidbrochart Here I'm assuming that a widget factory's `modelName` is always equivalent to the file type. This seems to hold for all the widget factories I've seen, and it seems to work at least for notebooks and text files. Is this a safe assumption? If not, how else can I retrieve the "current" file type from the created/opened document?